### PR TITLE
Create a simple executable script

### PR DESCRIPTION
**Pull request by nicokosi**
Document how to create a Unix-like executable script (which
still requires a Java interpreter).

### CODE REVIEW

**Comment by fbiville** [comment]
👍

File: make-exec.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```
Comment (fbiville) [inline make-exec.sh]: + `set -e` at the beginning
http://stackoverflow.com/questions/3474526/stop-on-first-error

File: make-exec.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+cat stub.sh target/hubstats-*-standalone.jar > $1 && chmod +x $1
```
Comment (fbiville) [inline make-exec.sh]: maybe include `lein uberjar` in the script?

File: make-exec.sh (hunk 1 above)
Comment (nicokosi) [inline make-exec.sh]: You're right, the script should fail-fast.

**Pull request closed without merging**